components/Ink/src/private/MDCSimpleInkGestureRecognizer.m

```diff
@@ -0,0 +1,71 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCSimpleInkGestureRecognizer.h"
+
+@implementation MDCSimpleInkGestureRecognizer
+
+- (instancetype)initWithTarget:(id)target action:(SEL)action {
+  self = [super initWithTarget:target action:action];
+  if (self) {
+    [self commonMDCSimpleInkGestureRecognizerInit];
```

Comment:
I only expect a common*Init method when there are multiple designated initializers for a class.

Since UIGestureRecognizer only has one, I would pull the content of commonInit into this method and get rid of commonInit.

Reply:
Done.